Javascript/Goodboy-Oldboy /script.js

```diff
@@ -0,0 +1,18 @@
+let dogYearBirth = 2014;
+let dogYearFuture = 2031;
+let shouldShowResultInDogYears = true;
+
+let dogAge;
```

Comment:
It's a good practice to define variables as a number or empty string, if you already know what they would be changed to. In this case:
```suggestion
let dogAge = 0;
```